stpsf/roman.py

```diff
@@ -682,10 +771,10 @@ def _load_detector_aberrations(self, path):
         path : string
             The path to the file containing detector aberrations.
         """
-        detectors = _load_wfi_detector_aberrations(path)
+        detectors_dict = _load_wfi_detector_aberrations(path)
         assert len(detectors.keys()) > 0
```

Comment:
I'm getting an error here.

This line should be `assert len(detectors_dict.keys()) > 0` instead

Reply:
Just changed in the new commit. I was prompted to make this change after noticing a name clash with the `detectors` imported at the top of the file.